source/components/dispatcher/dswload2.c

```diff
@@ -698,10 +698,14 @@ AcpiDsLoad2EndOp (
                 }
             }
 
-            AcpiExExitInterpreter ();
-            Status = AcpiEvInitializeRegion (
-                AcpiNsGetAttachedObject (Node), FALSE);
-            AcpiExEnterInterpreter ();
+            if (WalkState->PassNumber == ACPI_IMODE_EXECUTE &&
+                !(WalkState->ParseFlags & ACPI_PARSE_MODULE_LEVEL))
+            {
+                AcpiExExitInterpreter ();
+                Status = AcpiEvInitializeRegion (
+                    AcpiNsGetAttachedObject (Node));
+                AcpiExEnterInterpreter ();
+            }
```

Comment:
This is conflict against MLC support.
SystemMemory will become inaccessible during table loading.
I'm considering to fix this by splitting the _REG execution to a new position like UpdateGpes().